app/serializers/api/v1/offer_summary_serializer.rb

```diff
@@ -22,6 +22,22 @@ def include_messages?
       @options[:include_messages] == true
     end
 
+    def inventoried_package_count
+      object.inventory_packages.size
+    end
+
+    def inventoried_package_count__sql
+      "SELECT COUNT(*) FROM packages INNER JOIN items ON items.id = packages.item_id INNER JOIN offers ON offers.id = items.offer_id WHERE items.offer_id = 729 and inventory_number is not null"
```

Comment:
There's a hardcoded `items.offer_id = 729`

Reply:
Ahh.. I missed this while doing testing. That's embarrassing 🤦‍♂ .